src/main/minio.ts

```diff
@@ -3628,134 +3676,5 @@ Client.prototype.getObjectLegalHold=promisify(Client.prototype.getObjectLegalHol
 Client.prototype.composeObject = promisify(Client.prototype.composeObject)
 Client.prototype.selectObjectContent=promisify(Client.prototype.selectObjectContent)
 
-export class CopyConditions {
-  constructor() {
-    this.modified = ''
-    this.unmodified = ''
-    this.matchETag = ''
-    this.matchETagExcept = ''
-  }
-
-  setModified(date) {
-    if (!(date instanceof Date))
-      throw new TypeError('date must be of type Date')
-
-    this.modified = date.toUTCString()
-  }
-
-  setUnmodified(date) {
-    if (!(date instanceof Date))
-      throw new TypeError('date must be of type Date')
-
-    this.unmodified = date.toUTCString()
-  }
-
-  setMatchETag(etag) {
-    this.matchETag = etag
-  }
-
-  setMatchETagExcept(etag) {
-    this.matchETagExcept = etag
-  }
-}
-
-// Build PostPolicy object that can be signed by presignedPostPolicy
-export class PostPolicy {
```

Comment:
Also moved PostPolicy to a separate file, we'll re-export them from the minio.ts file later on.